tests/roles/dataplane_adoption/tasks/main.yaml

```diff
@@ -75,7 +75,7 @@
           ssh \
             -i {{ edpm_privatekey_path | default("/tmp/ansible_private_key") }} \
             {{ edpm_user }}@"${computes[$name]}" \
-            "echo $uuid > /var/lib/nova/compute_id"
+            "sudo echo $uuid > /var/lib/nova/compute_id"
```

Comment:
@archana203 I've added it here

Reply:
I still see permission denied with this:
[stack@undercloud-0 ~]$ ssh -i ~/.ssh/id_rsa stack@192.168.23.10 "sudo echo test > /var/lib/nova/compute_id"
bash: line 1: /var/lib/nova/compute_id: Permission denied

Reply:
@bogdando @gibizer @SeanMooney do you have any clue on how to make this work?
In our CI we use `root` so it works fine, but on RHEV we have the `stack` user and we are hitting this error

Reply:
As @sean-k-mooney have recommended for a similar case, consider using `edpm_user` role to create an `openstack` user (or stack?), then make sure when a task uses ``become:true`` (instead of sudo) it sets the user and group to ansible user, like this https://github.com/openstack-k8s-operators/edpm-ansible/blob/main/roles/edpm_nova/tasks/configure.yml#L3-L19

Reply:
something like this? https://github.com/openstack-k8s-operators/edpm-ansible/pull/534/files